<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tensor Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="simulateButton">
      <property name="text">
       <string>Simulate</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="exitButton">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="helpButton">
      <property name="text">
       <string>Help</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QProgressBar" name="simulationProgressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="emitterGroupBox">
      <property name="title">
       <string>Emitter</string>
      </property>
      <widget class="QRadioButton" name="emitterRecRadioButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>150</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Rectangular</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="emitterCylRadioButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>150</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Cyllindric</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="emitterElementsGroupBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>220</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Elements</string>
       </property>
       <widget class="QLineEdit" name="emitterElementsDepthLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QLabel" name="emitterElementsWidthLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>22</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>width</string>
        </property>
       </widget>
       <widget class="QLabel" name="emitterElementsDepthLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>42</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>depth</string>
        </property>
       </widget>
       <widget class="QLabel" name="emitterElementsHeightLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>62</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>height</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="emitterElementsWidthLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>22</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>elements</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="emitterElementsHeightLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>60</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>42</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>elements</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>62</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>elements</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="emitterSizeGroupBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>220</width>
         <height>91</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Size</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QLabel" name="emitterWidthMetersLabel">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>22</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>m</string>
        </property>
       </widget>
       <widget class="QLabel" name="emitterWidthLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>22</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>width</string>
        </property>
       </widget>
       <widget class="QLabel" name="emitterHeightMetersLabel">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>62</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>m</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="emitterHeightLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>60</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>1e-9</string>
        </property>
       </widget>
       <widget class="QLabel" name="emitterDepthLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>42</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>depth</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="emitterDepthLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>1e-9</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="emitterWidthLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>1e-9</string>
        </property>
       </widget>
       <widget class="QLabel" name="emitterHeightLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>62</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>height</string>
        </property>
       </widget>
       <widget class="QLabel" name="emitterDepthMetersLabel">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>42</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>m</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="emitterPositionGroupBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>220</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Position</string>
       </property>
       <property name="hidden" stdset="0">
        <bool>false</bool>
       </property>
       <widget class="QLineEdit" name="emitterYLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="emitterXLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>22</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
       <widget class="QLabel" name="emitterYLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>42</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>y</string>
        </property>
       </widget>
       <widget class="QLabel" name="emitterZLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>62</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>z</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="emitterXLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="emitterZLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>60</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </widget>
      <widget class="QComboBox" name="emitterComboBox">
       <property name="geometry">
        <rect>
         <x>165</x>
         <y>38</y>
         <width>63</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>xy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>yz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>xz</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="collectorGroupBox">
      <property name="title">
       <string>Collector</string>
      </property>
      <widget class="QRadioButton" name="collectorCylRadioButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>150</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Cyllindric</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="collectorElementsGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>220</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Elements</string>
       </property>
       <widget class="QLineEdit" name="collectorElementsDepthLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QLabel" name="collectorElementsWidthLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>22</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>width</string>
        </property>
       </widget>
       <widget class="QLabel" name="collectorElementsDepthLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>42</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>depth</string>
        </property>
       </widget>
       <widget class="QLabel" name="collectorElementsHeightLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>62</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>height</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="collectorElementsWidthLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_52">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>22</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>elements</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="collectorElementsHeightLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>60</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_53">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>42</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>elements</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_54">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>62</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>elements</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="collectorSizeGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>220</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Size</string>
       </property>
       <widget class="QLabel" name="collectorWidthMetersLabel">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>22</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>m</string>
        </property>
       </widget>
       <widget class="QLabel" name="collectorWidthLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>22</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>width</string>
        </property>
       </widget>
       <widget class="QLabel" name="collectorHeightMetersLabel">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>62</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>m</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="collectorHeightLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>60</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>1e-9</string>
        </property>
       </widget>
       <widget class="QLabel" name="collectorDepthLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>42</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>depth</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="collectorDepthLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>1e-9</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="collectorWidthLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>1e-9</string>
        </property>
       </widget>
       <widget class="QLabel" name="collectorHeightLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>62</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>height</string>
        </property>
       </widget>
       <widget class="QLabel" name="collectorDepthMetersLabel">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>42</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>m</string>
        </property>
       </widget>
      </widget>
      <widget class="QRadioButton" name="collectorRecRadioButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>150</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Rectangular</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="collectorPositionGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>220</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Position</string>
       </property>
       <widget class="QLineEdit" name="collectorYLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="collectorXLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>22</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
       <widget class="QLabel" name="collectorYLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>42</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>y</string>
        </property>
       </widget>
       <widget class="QLabel" name="collectorZLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>62</y>
          <width>62</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>z</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="collectorXLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="collectorZLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>60</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </widget>
      <widget class="QComboBox" name="collectorComboBox">
       <property name="geometry">
        <rect>
         <x>165</x>
         <y>38</y>
         <width>63</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>xy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>yz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>xz</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="settingsGroupBox">
      <property name="title">
       <string>Settings</string>
      </property>
      <widget class="QLineEdit" name="threadNumberLineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QLabel" name="threadsNumberLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>How many threads</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
   <zorder>simulationProgressBar</zorder>
   <zorder>emitterGroupBox</zorder>
   <zorder>exitButton</zorder>
   <zorder>helpButton</zorder>
   <zorder>simulateButton</zorder>
   <zorder>collectorGroupBox</zorder>
   <zorder>settingsGroupBox</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>emitterRecRadioButton</tabstop>
  <tabstop>emitterCylRadioButton</tabstop>
  <tabstop>emitterWidthLineEdit</tabstop>
  <tabstop>emitterDepthLineEdit</tabstop>
  <tabstop>emitterHeightLineEdit</tabstop>
  <tabstop>emitterXLineEdit</tabstop>
  <tabstop>emitterYLineEdit</tabstop>
  <tabstop>emitterZLineEdit</tabstop>
  <tabstop>emitterElementsWidthLineEdit</tabstop>
  <tabstop>emitterElementsDepthLineEdit</tabstop>
  <tabstop>emitterElementsHeightLineEdit</tabstop>
  <tabstop>collectorRecRadioButton</tabstop>
  <tabstop>collectorCylRadioButton</tabstop>
  <tabstop>collectorWidthLineEdit</tabstop>
  <tabstop>collectorDepthLineEdit</tabstop>
  <tabstop>collectorHeightLineEdit</tabstop>
  <tabstop>collectorXLineEdit</tabstop>
  <tabstop>collectorYLineEdit</tabstop>
  <tabstop>collectorZLineEdit</tabstop>
  <tabstop>collectorElementsWidthLineEdit</tabstop>
  <tabstop>collectorElementsDepthLineEdit</tabstop>
  <tabstop>collectorElementsHeightLineEdit</tabstop>
  <tabstop>threadNumberLineEdit</tabstop>
  <tabstop>simulateButton</tabstop>
  <tabstop>helpButton</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
